### PR TITLE
Element ID's for Smoke Tests

### PR DESCRIPTION
**Pull request by tombuildsstuff**
We're working on some Smoke Tests in an internal Build Tooling repo - which we'll probably end up adding to this project.

For now - this involves adding some Element ID's - so that we can target these with Selenium Element ID's. This might as well remain a WIP until we're happy enough with the Smoke Tests to run them properly.

cc @andyroyle / FYI @matteofigus

### CODE REVIEW

File: client/views/main.jade [hunk 1]
```diff
@@ -2,7 +2,7 @@ doctype html
 html(lang='en')
     head
         meta(name='viewport', content='width=device-width, initial-scale=1')
-        title Opentable Feature Toggles
+        title Hobknob Dashboard
```
Comment (andyroyle) [inline client/views/main.jade]: thought you'd sneak that one in did you? 😉

**Comment by stevejhiggs** [comment]
bring it up to date and I'll merge it

**Comment by chriscartlidge** [comment]
@tombuildsstuff Is this ready to be merged? It still has a [WIP] tag

**Comment by tombuildsstuff** [comment]
@chriscartlidge I've had to pause the smoke tests for the moment, I'll rebase this, then we should be able to merge it? Once we're happy with our smoke tests, we'll send a PR

**Pull request merged**